synapse/lib/aha.py

```diff
@@ -555,6 +644,128 @@ async def addAhaSvc(self, name, info, network=None):
         # mostly for testing...
         await self.fire('aha:svcadd', svcinfo=svcinfo)
 
+    def _getAhaName(self, name):
+        # the modern version of names is absolute or ...
+        if name.endswith('...'):
+            netw = self.conf.get('aha:network')
```

Comment:
Does this still need to handle the case where `aha:network` isn't set?

Reply:
I'll make it explicitly blow up :+1: